### DISCUSSION TITLE
migrating from on premise to hosted outline

### DISCUSSION THREAD

**Discussion by Freundschaft** [outline/outline#2618] [Self Hosting] [answered]
Hello, we are using outline self hosted version at the moment, but would consider migrating to the hosted version offered by outline.

Is there any way to import our existing outline database to the hosted version?

Thanks!

**Comment by tommoor** [comment, marked as answer]
Hi, for security we don't allow this however if your self hosted installation is uptodate you can export and import all text and image content.

Either export individual collections, or admins can export the entire wiki.

**Reply by accesstechnology-mike** [reply]
@Freundschaft did this work ok for you? I don't get any email  - I presume some error on my self-hosted setup. I can also only export half of the collections. 

@tommoor, am I right in thinking nested collections can't be exported?

**Reply by tommoor** [reply]
There isn't such a thing as nested collections, but nested documents should export without issue. It does depend on how old your installation is of course, there have been many bug fixes over the years.

**Reply by Freundschaft** [reply]
ok got it thanks!
